src/components/Pressable/Pressable.tsx

```diff
@@ -52,56 +54,64 @@ export default function Pressable(props: PressableProps) {
     [props.pressRetentionOffset]
   );
 
-  const pressGesture = useMemo(
-    () =>
-      Gesture.LongPress().onStart((event) => {
-        if (isPressedDown.current) {
-          props.onLongPress?.(adaptStateChangeEvent(event));
-          isPressCallbackEnabled.current = false;
-        }
-      }),
-    [props]
-  );
-
   const hoverInTimeout = useRef<number | null>(null);
   const hoverOutTimeout = useRef<number | null>(null);
 
   const hoverGesture = useMemo(
     () =>
       Gesture.Hover()
+        .manualActivation(true) // Stops Hover from blocking Native gesture activation on web
+        .cancelsTouchesInView(false)
         .onBegin((event) => {
           if (hoverOutTimeout.current) {
             clearTimeout(hoverOutTimeout.current);
           }
           if (props.delayHoverIn) {
             hoverInTimeout.current = setTimeout(
-              () => props.onHoverIn?.(adaptStateChangeEvent(event)),
+              () => props.onHoverIn?.(gestureToPressableEvent(event)),
               props.delayHoverIn
             );
             return;
           }
-          props.onHoverIn?.(adaptStateChangeEvent(event));
+          props.onHoverIn?.(gestureToPressableEvent(event));
         })
-        .onEnd((event) => {
+        .onFinalize((event) => {
           if (hoverInTimeout.current) {
             clearTimeout(hoverInTimeout.current);
           }
           if (props.delayHoverOut) {
             hoverOutTimeout.current = setTimeout(
-              () => props.onHoverOut?.(adaptStateChangeEvent(event)),
+              () => props.onHoverOut?.(gestureToPressableEvent(event)),
               props.delayHoverOut
             );
             return;
           }
-          props.onHoverOut?.(adaptStateChangeEvent(event));
+          props.onHoverOut?.(gestureToPressableEvent(event));
         }),
     [props]
   );
 
   const pressDelayTimeoutRef = useRef<number | null>(null);
+  const propagationGreenLight = useRef<boolean>(false);
+
+  // iOS only, propagationGreenLight setting occurs after other checks when in scroll-view
```

Comment:
No, it's intended for `awaitingEventPayload`.

But I see it is very poorly worded which makes it look like it's misplaced,
maybe a comment like this would work better?

```suggestion
  // iOS only, due to variable flow of gestures on iOS, their events sometimes have to be saved for later use
```

Reply:
What about
```suggestion
  // For iOS only: due to the varying flow of gestures, events sometimes have to be saved for later use
```

or simply:

```suggestion
  // iOS only: due to the varying flow of gestures, events sometimes have to be saved for later use```

Reply:
I like the last one the most 👍